vector/src/main/java/fr/gouv/tchap/adapters/TchapContactAdapter.java

```diff
@@ -385,20 +385,46 @@ private void populateViews(final ParticipantAdapterItem participant, final int p
             }
 
             // show the partipant different depending on priority
-            if (!participant.isViewedInPriority()){
+            if (participant.isMatrixUser()){
+                vContactAvatar.clearColorFilter();
+                vContactDomain.setVisibility(View.VISIBLE);
+                vContactName.setTypeface(null, Typeface.BOLD);
+                vContactName.setText(DinsicUtils.getNameFromDisplayName(participant.mDisplayName));
+            } else {
                 final int semiTransparentGrey = Color.argb(155, 185, 185, 185);
                 vContactAvatar.setColorFilter(semiTransparentGrey);
+                vContactDomain.setVisibility(View.GONE);
                 vContactName.setTypeface(null, Typeface.ITALIC);
-            }
-            else{
-                vContactAvatar.clearColorFilter();
-                vContactName.setTypeface(null, Typeface.BOLD);
+                vContactName.setText(participant.mDisplayName);
             }
 
+            // display the participant's avatar
             participant.displayAvatar(mSession, vContactAvatar);
 
-            vContactName.setText(DinsicUtils.getNameFromDisplayName(participant.mDisplayName));
-            vContactDomain.setText(DinsicUtils.getDomainFromDisplayName(participant.mDisplayName));
+            // display the participant's domain
```

Comment:
Move the participant's domain handling under the condition "if (participant.isMatrixUser())"